src/main/java/sg/ncl/MainController.java

```diff
@@ -3451,14 +3458,16 @@ private Realization invokeAndExtractRealization(String teamName, Long id) {
                 log.warn("error in retrieving realization for team: {}, realization: {}", teamName, id);
                 return getCleanRealization();
             } else {
+                // will throw JSONException if the format return by sio is not a valid JSOn format
+                // will occur if the realization details are still in the old format
                 return extractRealization(responseBody);
             }
-        } catch (IOException e) {
+        } catch (IOException | JSONException e) {
             return getCleanRealization();
         }
     }
 
-    private Realization extractRealization(String json) {
+    private Realization extractRealization(String json) throws JSONException {
```

Comment:
![MINOR](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/severity-minor.png) Remove the declaration of thrown exception 'org.json.JSONException' which is a runtime exception. [![rule](https://raw.githubusercontent.com/SonarCommunity/sonar-github/master/images/rule.png)](https://sonarqube.com/coding_rules#rule_key=squid%3ARedundantThrowsDeclarationCheck)